src/assets/sampleBobaAPI.tsx

```diff
@@ -5,8 +5,8 @@ export interface BobaProps {
   description: string;
   price: number;
   imageLink: string;
-  defaultOptions: {
-    milk: string;
+  options: {
+    milk: Array<{ name: string }>;
```

Comment:
Should just be object. We don't have multiple milk objects in array like toppings array